test/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclarationTest.java

```diff
@@ -860,7 +860,7 @@ public void testDoWhileForOfCapturedLet() {
   }
 
   // https://github.com/google/closure-compiler/issues/1124
```

Comment:
which you already added a comment to, and I missed it because github displays comments in the lightest of light grays. Nevermind then, thanks.